### ISSUE TITLE
Installing Refprop on Mac M1 (OSX with Office 365)

### ISSUE THREAD

**Issue by Vincen-Jo** [usnistgov/REFPROP-cmake#74]
Hi,

I visited this place again after a long time, and it seems that all of you who are capable here have found a way to work on the M1 MacBook.
So I tried it again. The installation environment is as follows:
Anaconda 64-Bit(M1) Graphical Installer has been installed and brew gfortran has been installed.
However, the following error occurred:
<img width="1096" alt="2022-12-04 16 02 45" src="https://user-images.githubusercontent.com/80340673/205478750-a91d0cfc-9f79-4552-9bc8-8e6f00a61c58.png">

**Comment by ianhbell** [comment]
There’s no error yet, that all looks fine, you need to build now.

**Comment by Vincen-Jo** [comment]
When I enter cmake --build . and run it, the following error occurs.

<img width="960" alt="2022-12-06 14 53 28" src="https://user-images.githubusercontent.com/80340673/205829650-c7f1c2ba-0e1e-4162-9a70-2e456d601ca3.png">

**Comment by ianhbell** [comment]
Sounds like you are building for two different architectures.  Let me see how I got this working on my M1.

**Comment by ianhbell** [comment]
Can you comment out these lines in ``CMakeLists.txt`` and try again:

<img width="853" alt="image" src="https://user-images.githubusercontent.com/11966765/205929597-e9d04826-1578-4a01-a95d-1d39539b5e3a.png">

**Comment by Vincen-Jo** [comment]
The results are as follows.
Did it go well?

<img width="814" alt="2022-12-06 23 11 06" src="https://user-images.githubusercontent.com/80340673/205934625-f44fd199-d9e7-4ba3-a404-bb500907e39b.png">

**Comment by ianhbell** [comment]
Yes, looks good. I guess I will have to add some more flags to deal with M1 building, as I suspect this will continue to be a problem.

**Comment by Vincen-Jo** [comment]
Thank you very much.
I will do the next step myself.

**Comment by ianhbell** [comment]
You're welcome

**Comment by Vincen-Jo** [comment]
I added the REFPROP.XLA file to add-in, but following error occurs.

<img width="488" alt="2022-12-07 00 10 34" src="https://user-images.githubusercontent.com/80340673/205949910-4026b377-4aa3-4de7-a2f6-7915aaa58cdd.png">
<img width="1501" alt="2022-12-07 00 10 56" src="https://user-images.githubusercontent.com/80340673/205949938-5c602283-cf64-4c11-83cb-2c837a119546.png">
<img width="1527" alt="2022-12-07 00 11 52" src="https://user-images.githubusercontent.com/80340673/205949976-846ce7d4-308a-4c48-bd17-af089419b39f.png">

**Comment by ianhbell** [comment]
Was that error message returned by the Add-in? I haven't seen that before.

**Comment by Vincen-Jo** [comment]
No error message occurred when adding to Add-in.
I succeeded at once on Windows, but it is hard to try on MacOS. ^_^

**Comment by ianhbell** [comment]
Yes, getting this working on Mac is much trickier, I don't have any other ideas to try.  Personally I don't use Excel and use other tools instead.

**Comment by Vincen-Jo** [comment]
Okay.
I will try again with other tools.
Have a good day!
